src/grunt/helper.js

```diff
@@ -111,34 +111,41 @@ ns.init = function (grunt) {
                 reporters: ['progress', 'coverage'],
 
                 coverageReporter: {
-                    reporters: [{
-                        type: 'text-summary'
-                    }, {
-                        type: 'html',
-                        dir: path.join(process.cwd(), 'coverage'),
-                    }],
+                    reporters: [
+                        {
+                            type: 'text-summary',
+                        },
+                        {
+                            type: 'html',
+                            dir: path.join(process.cwd(), 'coverage'),
+                        },
+                    ],
```

Comment:
While I like the shorter format I'm happy to sacrifice it for enforced trailing commas. :+1: